components/common.js

```diff
@@ -0,0 +1,52 @@
+import {
+  Indent,
+  IndentationTypes,
+  withIndendation,
+} from "@asyncapi/generator-react-sdk";
+
+export function HTML({ childrenContent }) {
+  return `
+<!DOCTYPE html>
+<html lang="en">
+${childrenContent}
+</html>
+`;
+}
+
+export function Head({ title, cssLinks = [], styleContent = [],base = "" }) {
+  const links = cssLinks
+    .map((link) => `<link rel="stylesheet" href="${link}">\n`);
+
+  const content = `
+<head>
+  <meta charset="utf-8">
+  ${base && `<base href=${base}>`}
+  <title>${title}</title>
+${withIndendation(links, 2, IndentationTypes.SPACES)}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  ${(styleContent.length > 0) ? `<style>\n${styleContent[0]}\n${styleContent[1]}</style>` : ""}
+  ${(cssLinks.length > 0) ? `\n<link href=${cssLinks[0]}>\n<link href=${cssLinks[1]}>\n` : ""}
```

Comment:
You have performed links and saved in the `links` array. Why you do it again in the `28` line? :)

Reply:
basically, we have `cssLinks `array and `styleContent` array so if I am thinking correctly we will only add `styleContent` when the array is not empty and the same goes for `cssLinks` so according to me we have to check them separately. :)

Reply:
Yeah, you understand it correctly, but please remove:

```
  const links = cssLinks
    .map((link) => `<link rel="stylesheet" href="${link}">\n`);
```

because currently it's unnecessary :)

Reply:
Done! :)